slasher/beaconclient/receivers.go

```diff
@@ -33,6 +39,13 @@ func (bs *Service) receiveBlocks(ctx context.Context) {
 			log.WithError(ctx.Err()).Error("Context canceled - shutting down blocks receiver")
 			return
 		}
+		if err != nil && strings.Contains(err.Error(), "Context canceled") {
```

Comment:
Can we maybe move the error text to a global var?

Reply:
changed to context.Canceled.Error()

Reply:
thanks